packages/form-generator/src/codegen_graphql_form_generation_result.ts

```diff
@@ -11,7 +11,7 @@ export class CodegenGraphqlFormGeneratorResult
   /**
    * Creates a CodegenGraphqlFormGeneratorResponse
    */
-  constructor(private readonly fileNameComponentMap: Record<string, string>) {}
+  constructor(public readonly fileNameComponentMap: Record<string, string>) {}
```

Comment:
This field should not be made public in order to satisfy the test. The public interface already allows a way to make this assertion.

Reply:
Thanks will fix